src/main/java/frc/robot/RobotContainer.java

```diff
@@ -32,10 +52,18 @@
  */
 public class RobotContainer {
 
+    private static final Limelight LL_FR = new Limelight(LimelightType.LL4, "limelight-fr", true, true);
+    private static final Limelight LL_FL = new Limelight(LimelightType.LL4, "limelight-fl", true, true);
+    private static final Limelight LL_BR = new Limelight(LimelightType.LL4, "limelight-br", true, true);
+    private static final Limelight LL_BL = new Limelight(LimelightType.LL4, "limelight-bl", true, true);
+
+    public static final LimelightContainer LLContainer = new LimelightContainer(LL_FR, LL_FL, LL_BR, LL_BL);
+
     private final CommandXboxController driverXbox = new CommandXboxController(
             ControllerConstants.DRIVER_CONTROLLER_PORT);
     private final CommandXboxController operatorXbox = new CommandXboxController(
             ControllerConstants.OPERATOR_CONTROLLER_PORT);
+    private final CommandXboxController debugXboxController = new CommandXboxController(3);
```

Comment:
If this is just for debugging it should be removed.
```suggestion
[-]    private final CommandXboxController debugXboxController = new CommandXboxController(3);
```